common/protob/messages-cardano.proto

```diff
@@ -144,6 +146,17 @@ message CardanoSignTx {
         // repeated uint32 address_n = 2;                               // moved to address_parameters
         optional uint64 amount = 3;                                     // amount to spend
         optional CardanoAddressParametersType address_parameters = 4;   // parameters used to derive the address
+        repeated CardanoTokenGroupType token_bundle = 5;                // custom assets - added in mary
+    }
+
+    message CardanoTokenGroupType {
```

Comment:
It's confusing to me that the types and fields mention tokens and the comments mention assets. Wouldn't `CardanoAssetGroupType` and `CardanoAssetAmountType` be better names? Or is there some subtle distinction? I've checked the documentation and it hasn't made it any clearer. So perhaps it's just worth to give it a second thought.